Sources/PointFree/Pricing.swift

```diff
@@ -536,28 +540,28 @@ private let selectors = (
     CssSelector.id("tab1")
   ),
   content: (
-    CssSelector.id("content0"),
-    CssSelector.id("content1")
+    CssSelector.class("content0"),
```

Comment:
Changed these to use `class` selectors so we can hide/show things more granularly.

Reply:
ah nice, that works! cool that we have the screenshot tests that execute JS to switch tabs!